liri.js

```diff
@@ -0,0 +1,173 @@
+//npm packages...
+var fs = require("fs");
+
+var keys = require("./keys.js");
+
+var request = require("request");
+
+var Twitter = require('twitter');
+
+var Spotify = require('node-spotify-api');
+
+var fs = require("fs");
+
+var inquirer = require("inquirer");
+
+// Created a series of questions that will eventually determine what function to call and what data to pass into it.
+inquirer.prompt([
+
+  {
+    type: "input",
+    name: "name",
+    message: "What is your name?"
+  },
+
+  {
+    type: "list",
+    name: "opperator",
+    message: "What would you like me to do for you?",
+    choices: ["Show your tweets", "Spotify a song", "IMDB a movie", "Do what ever it says"]
+  }
+
+]).then(function(answers) {
+
+  if (answers.opperator == "Spotify a song") {
+    inquirer.prompt([
+      {
+        type: "input",
+        name: "songName",
+        message: "What song?"
+      }
+    ]).then(function(answers) {
+      songName = answers.songName;
+      spotify();
+    })
+  }
+  else if (answers.opperator == "Show your tweets") {
+    twitter();
+  }
+  else if (answers.opperator == "Do what ever it says") {
+    doWhatItSays();
+  }
+  else if (answers.opperator == "IMDB a movie") {
+    inquirer.prompt([
+      {
+        type: "input",
+        name: "movieName",
+        message: "What movie?"
+      }
+    ]).then(function(answers) {
+      movieName = answers.movieName;
+      imdb();
+    })
+  }
+
+});
+
+//Function that retrieves the last 20 tweets from my twitter account
+function twitter() {
+	var client = new Twitter({
+	  consumer_key: keys.twitterKeys.consumer_key,
+	  consumer_secret: keys.twitterKeys.consumer_secret,
+	  access_token_key: keys.twitterKeys.access_token_key,
+	  access_token_secret: keys.twitterKeys.access_token_secret
+	});
+
+	var params = {
+		screen_name: 'ChristinaMatson',
+		count: 20
+	};
+	client.get('statuses/user_timeline', params, function(error, tweets, response) {
+	  if (!error) {
```

Comment:
It is better practice to test for errors so that you can handle them appropriately instead of handling not errors.